src/AltinnCore/Designer/Controllers/ReposController.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Designer.Controllers
+{
+    /// <summary>
+    ///  Some shit
```

Comment:
It would be nice if this is removed and a meaningful summary is added about repos

Reply:
This is outdated, see newest commit.